pkg/apis/config/defaults.go

```diff
@@ -128,3 +156,36 @@ func (d *Defaults) GetBrokerClass(ns string) (string, error) {
 	}
 	return "", errors.New("Defaults for Broker Configurations have not been set up.")
 }
+
+// getClassConfig get the corresponding class Config in multiple classes
+func getClassConfig(d *Defaults, cb *ClassAndBrokerConfig) *BrokerConfig {
+	if cb == nil || cb.BrokerClass == "" {
+		return nil
+	}
+	if bCSpec := getBrokerClasses(d); bCSpec != nil {
+		bConfig := matchBrokerClass(cb.BrokerClass, bCSpec)
```

Comment:
here, we are always using `ClusterDefault.BrokerClass` but if I create a Broker with an already set Broker class, we won't chose the correct configuration

Reply:
> here, we are always using cb.BrokerClass which is equal to `ClusterDefault.BrokerClass` but if I create a Broker with an already set Broker class, we won't chose the correct configuration
But the brokerclass what we set is in cluster level right? We always need to set a `BrokerClass` in `ClusterDefault`, and then we choose the corresponding `BrokerConfig`